sdk/src/main/java/com/mapbox/maps/renderer/RendererError.kt

```diff
@@ -0,0 +1,100 @@
+package com.mapbox.maps.renderer
+
+import com.mapbox.maps.MapView
+import javax.microedition.khronos.egl.EGL10
+import javax.microedition.khronos.egl.EGL11
+
+/**
+ * Typed enum wrapping up all the EGL error codes represented as ints.
+ */
+enum class RendererError {
```

Comment:
Does it cover all the error types? Maybe we should just propagate the error code as Integers to the user?

Reply:
it does cover all EGL errors based on the [spec](https://www.khronos.org/files/egl-1-4-quick-reference-card.pdf). Even this is imho a bit of an overkill but decided that better more than less 😄